nautobot/extras/api/serializers.py

```diff
@@ -946,7 +954,18 @@ class TeamSerializer(NautobotModelSerializer):
     class Meta:
         model = Team
         fields = "__all__"
-        extra_kwargs = {"contacts": {"required": False}}
+        extra_kwargs = {
+            "contacts": {"required": False},
+            "email": {"default": ""},
+            "phone": {"default": ""},
+        }
+        # https://www.django-rest-framework.org/api-guide/validators/#optional-fields
+        validators = []
+
+    def validate(self, data):
+        validator = UniqueTogetherValidator(queryset=Team.objects.all(), fields=("name", "phone", "email"))
```

Comment:
Can this go into the `Meta.validators` above?

Reply:
Or is that what causes the problem we're experiencing?

Reply:
I think that's what causes the problem, but *maybe* the addition of `default` values here may also address the problem. I'll try it out and report back.

Reply:
Well, bummer. This does match other serializers where we had to do this though